src/rai_extensions/rai_open_set_vision/rai_open_set_vision/vision_markup/boxer.py

```diff
@@ -61,12 +61,12 @@ def to_detection_msg(
 class GDBoxer:
     def __init__(
         self,
-        weight_path: (
-            str | PathLike
-        ) = "./src/rai_extensions/rai_grounding_dino/GroundingDINO/weights/groundingdino_swint_ogc.pth",
+        weight_path: str | PathLike,
         use_cuda: bool = True,
     ):
-        self.cfg_path = __file__.replace("boxer.py", "config.py")
+        self.cfg_path = __file__.replace(
+            "vision_markup/boxer.py", "configs/gdino_config.py"
+        )
```

Comment:
What is going on here?

Reply:
The model constructor expects a path to the config file. The path is relative to the directory of the `vision_markup/boxer.py`. This takes the path to `__file__` and takes the relative one.

Reply:
Maybe get_package_share_direcotry could help?